torch/csrc/jit/passes/annotate_warns.cpp

```diff
@@ -0,0 +1,27 @@
+#include <torch/csrc/jit/passes/annotate_warns.h>
+
+namespace torch {
+namespace jit {
+
+void AnnotateWarns(Block* b) {
+  static int64_t idx = 0;
```

Comment:
you probably need to make it atomic as compilation might happen concurrently from different threads

Reply:
Ended up removing the idx per your suggestion.